<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
    <item row="4" column="0">
     <widget class="QPushButton" name="button_test_draw_city">
      <property name="toolTip">
       <string>testet</string>
      </property>
      <property name="text">
       <string>Test Draw City</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEdit_teste_was">
      <property name="placeholderText">
       <string>Ihre Eingabe</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_eingabe">
      <property name="text">
       <string>Ihre Eingabe</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton_test_map_add_draw">
      <property name="toolTip">
       <string>testet</string>
      </property>
      <property name="text">
       <string>Test map, add, draw</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="pushButton_test_dijkstra">
      <property name="toolTip">
       <string>Testet den Dijkstra-Algorithmus</string>
      </property>
      <property name="text">
       <string>Test Dijkstra</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="pushButton_add_extern_map">
      <property name="toolTip">
       <string>Lässt Sie eine Stadt per Datei-Browser von Ihrer Festplatte einbinden</string>
      </property>
      <property name="text">
       <string>Add Extern Map</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="18">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_teste_was">
      <property name="toolTip">
       <string>Testet, ob Ihre Eingabe eine Zahl oder ein Text ist</string>
      </property>
      <property name="text">
       <string>Teste was</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="pushButton_test_abstract_map">
      <property name="toolTip">
       <string>Testet die Methoden der Klasse abstractmap</string>
      </property>
      <property name="text">
       <string>Test Abstract Map</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox_hide_show">
      <property name="toolTip">
       <string>versteckt/zeigt die unwichtigen Buttons</string>
      </property>
      <property name="text">
       <string>zeige Test-Buttons</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QPushButton" name="pushButton_add_city">
      <property name="toolTip">
       <string>lässt Sie eine Stadt hinzufügen</string>
      </property>
      <property name="text">
       <string>Add City</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="pushButton_test_draw_street">
      <property name="toolTip">
       <string>Testet, ob Straßen gezeichnet werden können</string>
      </property>
      <property name="text">
       <string>Test Draw Street</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QPushButton" name="pushButton_fill_map">
      <property name="toolTip">
       <string>Füllt die Grafik mit der aktuell eingebunden Karte</string>
      </property>
      <property name="text">
       <string>Fill Map</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="pushButton_test_add_street">
      <property name="toolTip">
       <string>Testet, ob Straßen hinzugefügt werden können</string>
      </property>
      <property name="text">
       <string>Test add Street</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QPushButton" name="pushButton_free_search">
      <property name="toolTip">
       <string>Lässt Sie zwei Städte eingeben und zeigt Ihnen dann den kürzesten Weg zwischen beiden</string>
      </property>
      <property name="text">
       <string>Free Search</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QPushButton" name="pushButton_add_street">
      <property name="toolTip">
       <string>Lässt Sie eine Straße hinzufügen</string>
      </property>
      <property name="text">
       <string>Add Street</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuScene">
    <property name="title">
     <string>Scene</string>
    </property>
    <addaction name="actionClear_Scene"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuScene"/>
   <addaction name="menuInfo"/>
  </widget>
  <action name="actionlol">
   <property name="text">
    <string>lol</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="actionClear_Scene">
   <property name="text">
    <string>Clear Scene</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
